src/targets/gpu/fuse_mlir.cpp

```diff
@@ -416,14 +416,15 @@ void fuse_mlir::apply(module_pass_manager& mpm) const
     mlir_mode mode =
         (enabled(MIGRAPHX_ENABLE_EXTRA_MLIR{}) or enable_extra) ? mlir_mode::fast : mlir_mode::none;
 
-    match::find_matches(mpm,
-                        find_mlir_fused_ops{.conv_mode = get_mode("fused", mlir_mode::fast),
-                                            .dot_mode  = get_mode("fused", mode)});
+    match::find_matches(
+        mpm,
+        find_mlir_fused_ops{.conv_mode = get_mode("fused", mlir_mode::fast),
+                            .dot_mode  = is_navi ? mlir_mode::fast : get_mode("fused", mode)});
 
     match::find_matches(
         mpm,
-        find_mlir_standalone_convolution_op{get_mode("convolution", mlir_mode::int8)},
-        find_mlir_standalone_dot_op{get_mode("dot", mlir_mode::none)});
+        find_mlir_standalone_convolution_op{get_mode("convolution", mlir_mode::fast)},
+        find_mlir_standalone_dot_op{is_navi ? mlir_mode::fast : get_mode("dot", mlir_mode::none)});
```

Comment:
This should probably be `get_mode("dot", mode)` so we use MLIR for dot fusions when using the `MIGRAPHX_ENABLE_EXTRA_MLIR` setting.

Reply:
Sorry thats not the case. This is standalone.